python/lsst/ts/ofc/ofc.py

```diff
@@ -251,7 +251,7 @@ def get_all_corrections(self):
         return corrections
 
     def get_correction(self, dof_comp):
-        """Get the hexapod correction.
+        """Get the correction.
```

Comment:
how about?

> Get correction for specified component.

Reply:
How about also indicating whether this is aggregated correction or just what is determined from the last visit?

Reply:
Done in the commit: e64d8ae. Thanks!